### ISSUE TITLE
Is there a URI for the OpenCTI API for testing?

### ISSUE THREAD

**Issue by smclinden** [OpenCTI-Platform/connectors#77]
Please replace every line in curly brackets { like this } with an appropriate answer, and remove this line.

## Description

All of the connectors from this repot (other than the import and export) fail to connect to the API but other services are running

## Environment

1. OS: Ubuntu 18:04
2. OpenCTI version: 3.1.0
3. OpenCTI client: frontemd

## Reproducible Steps

Steps to create the smallest reproducible scenario:

Added the connector container to the docker-compose.yml file and restart the stack. The connectors are active and OpenCTI appears to be working but none of the connectors can contact the API.

## Expected Output

I expect that the connectors to MISP/VT/COVID would work.

## Actual Output

OpenCTI API is not reachable. Waiting for OpenCTI API to start or check your configuration...
 
## Additional information

None

**Comment by smclinden** [comment]
Not hearing anything back, I decided to experiment. I changed the port for OpenCTI from 8080 to 4000 because it was suggested somewhere else.

I, then, cloned "connectors" and edited the config.yml (copied from the sample) to match the entries in the docker-compose.yml for OpenCTI.

I, also, added "RUN apk update" to the Dockerfile (to get rid of some warnings) and did a "docker build --tag connector:mytag ." and pointed the docker-compose.yml to my locally build container.

Voila, everything worked.

Initially, I tried the Docker images because I thought that it would save me some time. But since I tool all of the other dependencies out and ran them native on the host, it seems to me that this is the least frustrating way to go.

Hopefully this helps someone else.
